scripts/partition_manager/partition_manager.rst

```diff
@@ -812,6 +812,37 @@ You can add or remove partitions as described in the following sections.
 .. note::
   If the static configuration contains an entry for the ``app`` partition, this entry is ignored.
 
+.. _ug_pm_config_dynamic:
+
+Configuring dynamic partitions
+==============================
+
+For child images that use dynamic partition sizing, the Partition Manager allows you to use the ``CONFIG_PM_PARTITION_SIZE_<CHILD_IMAGE>`` Kconfig option to adjust the partition size.
+This is particularly useful when you want to optimize the memory usage of MCUboot to the smallest possible size by enabling cryptographic functionalities.
+
+.. note::
+   The ``CONFIG_PM_PARTITION_SIZE_<CHILD_IMAGE>`` option should be set in the child image's configuration to match the reduced size.
```

Comment:
```suggestion
   To match the reduced size, you need to set the ``CONFIG_PM_PARTITION_SIZE_<CHILD_IMAGE>`` Kconfig option in the child image's configuration.
```

Reply:
+all CONFIG_PM_PARTITION_SIZE options should probably render as Kconfig options, though the option itself doesn't appear in the Kconfig search